### DISCUSSION TITLE
Clerk Error

### DISCUSSION THREAD

**Discussion by ceowapp** [clerk/javascript#3496] [Support request - DO NOT OPEN HERE] [answered]
./node_modules/@clerk/shared/dist/react/index.mjs
Attempted import error: 'SWRConfig' is not exported from 'swr' (imported as 'SWRConfig').
![clerk_error](https://github.com/clerk/javascript/assets/128037213/2ac50411-8712-4cb3-897a-834680c7ecd7)

I was installing @clerk/nextjs and @clerk/react but got this error. I can't seem to make it work. I was wondering whether this error is fixed or not. Thank in advance.

**Comment by anish3333** [comment, marked as answer]
@ceowapp hey man did you solved the error??

**Reply by code-andy-stackbaker** [reply]
I am still facing this issue today. Did you get any help from somewhere?

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
